castervoice/lib/settings.py

```diff
@@ -192,6 +202,11 @@ def _find_natspeak():
         "winword": True,
         "wsr": True,
     },
+    
+    # sikuli settings
+    "sikuli": {
+        "version": ""
```

Comment:
It's debatable where the paths should go. Do they go in the Sikuli section because they're for Sikuli or in the paths section because they're paths? I do at agree though about the "enabled" property.